### PR TITLE
update the renamed fields

### PR DESCRIPTION
**Pull request by ianzhang366**
Signed-off-by: Ian Zhang <izhang@redhat.com>

### CODE REVIEW

File: pkg/templates/crds/hypershift/cluster.open-cluster-management.io_hypershiftdeployments.yaml [hunk 1]
```diff
@@ -1041,6 +1041,17 @@ spec:
                 - services
                 - sshKey
                 type: object
+              hostingManagedCluster:
```
Comment (zhujian7) [inline pkg/templates/crds/hypershift/cluster.open-cluster-management.io_hypershiftdeployments.yaml:1044]: should this be `hostingCluster`?

Comment (ianzhang366) [reply pkg/templates/crds/hypershift/cluster.open-cluster-management.io_hypershiftdeployments.yaml]: done

**Comment by ianzhang366** [comment]
@jnpacker can you please help with this one?

**Comment by ianzhang366** [comment]
yup this is ready @JakobGray

**Pull request merged**